modules/gapi/include/opencv2/gapi/s11n.hpp

```diff
@@ -42,6 +44,7 @@ T deserialize(const std::vector<char> &p);
 GAPI_EXPORTS std::vector<char> serialize(const cv::GCompileArgs&);
 GAPI_EXPORTS std::vector<char> serialize(const cv::GMetaArgs&);
 GAPI_EXPORTS std::vector<char> serialize(const cv::GRunArgs&);
+GAPI_EXPORTS std::vector<char> serialize(const std::vector<std::string>&);
```

Comment:
I don't think this should be generic (at least for now), but feel free to disagree